geoportal/c2cgeoportal_geoportal/scaffolds/create/print/print-apps/+package+/config.yaml.tmpl

```diff
@@ -74,6 +74,7 @@ templates:
             headers:
               - Referer
               - X-Request-ID
+              - Forwarded
```

Comment:
I'm not certain that we should add it in the default template ...

Reply:
It comes from https://github.com/camptocamp/c2cgeoportal/commit/4c5ec725bf401aec7322a1fe686ecc8f0401ea6c

Also, I've errors in the test because self.request.client_addr is None. Any idea ?

Reply:
For me, if we change the default scaffold of a stable release, it should be for a good reason for standard users.
In the commit, you point it was done for the intranet user detection, who was introduced in GMF 2.5...

Reply:
In version 2.4 we use a version of c2cwsgiutils who's didn't have c2cwsgiutils/client_info.py information...

I don't know what's the impact, but we use an old version of Pyramid...